lib/registries/README.md

```diff
@@ -25,3 +25,36 @@
+                                                                                                                                                                             │                                               ▼                                                  │
+                                                                                                               ┌───────────────────────┐                         ┌───────────────────────┐                       ┌───────────────────────┐                          ┌───────────────────────┐
+                                                                                                               │     Registration      │                         │     Registration      │                       │     Registration      │          Embargo         │     Registration      │
+                                                                                      ┌────── Embargoed ──────▶│      (Archiving)      │─ Archival Finished ────▶│  (Pending Approval)   │── Admins Approved ───▶│      (Embargoed)      │─────   Ended Early   ───▶│  (Pending Early End)  │
+                                                                                      │                        │      (Embargoed)      │                         │      (Embargoed)      │                       │                       │                          │                       │
+                                                                                      │                        └───────────────────────┘                         └───────────────────────┘                       └───────────────────────┘                          └───────────────────────┘
+                         Approval Not                                                 │                                                                                                                                Embargo Date                                              │
+                           Required    ┌─────────────────────┐                        │                                                                                                                                   Passes                                                 │
+                          Or Ommited   │ Draft Registration  │ Registered             │                                                                                                                                      │                                                   │
+                        ┌─────────────▶│     (Approved)      │────────────────────────┤                                                                                                                                      ├────────────────  Admins Approve   ────────────────┘
```

Comment:
It might be worth adding a line from the Draft Registration boxes that you can delete the draft registration. Similarly, can embargoed registrations be deleted rather than withdrawn?

Reply:
An embargo may be ended early (made public), or withdrawn while private, but not deleted.